internal/engine/compiler/impl_amd64.go

```diff
@@ -103,6 +103,7 @@ func newAmd64Compiler() compiler {
 	c := &amd64Compiler{
 		assembler:     amd64.NewAssembler(),
 		locationStack: newRuntimeValueLocationStack(),
+		cpuFeatures:   loadCpuFeatureFlags(),
```

Comment:
I think we should get this in global variable (e.g. `platform.CPUFeature`) and set it in `init` function of platform package. That's because the CPU features must be same across the process and we don't need to get multiple times